backend/dbscripts/thunderdb/postgres.sql

```diff
@@ -1,13 +1,72 @@
+-- Table to store Organizations
+CREATE TABLE ORGANIZATION (
+    ID              INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    ORGANIZATION_ID VARCHAR(36) UNIQUE NOT NULL,
+    NAME            TEXT               NOT NULL,
```

Comment:
Let's use VARCHAR with a predefined length instead of TEXT.
Check other places as well

Reply:
Fixed with https://github.com/asgardeo/thunder/compare/4c5372fa2836a57d03c012108b3cafc74abb909b..73946306ccf2e751a0829d5c8c47baf76a871d36